### ISSUE TITLE
Electrical Busses not powered correctly

### ISSUE THREAD

**Issue by heclak** [flybywiresim/aircraft#9629]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-12-04T11:01:22+00:00",
    "ref": "refs/heads/master",
    "sha": "2927b71d4ba68d65d17830fef7703a88d6d794cb",
    "actor": "Saschl",
    "event_name": "manual",
    "pretty_release_name": "master:2927b71d",
    "version": "a380x-v0.13.0-dev.2927b71"
}
```

### Describe the bug

_This bug is isolated to MSFS2024_

The electrical power logic appears to having some strange behaviour. 

1. The bus init state do not match between 2020 and 2024. In 2024, all the busses in the infinitbat are connected by default.

Init State of 2020
![image](https://github.com/user-attachments/assets/96d6e7c4-80eb-455e-807d-2f2e9759538a)

Init state of 2024
![image](https://github.com/user-attachments/assets/99f46f0c-f510-4073-8e90-3085b5fda3f5)

**Odd logic of the ESS BAT switch**
When using the ESS BAT switch, the state of the DC_ESS_BUS and AC_ESS_BUS is only toggled when the ESS BAT switch is in the ON position. This is instead of following the ON-OFF state of the ESS BAT switch. Thus two clicks (OFF->ON) are required to toggle the state of the ESS BAT. Therefore, it is possible to create a situation where the 

![image](https://github.com/user-attachments/assets/cc5b61a3-bf08-47be-99aa-a67029236121)

![image](https://github.com/user-attachments/assets/9b4a0e48-779f-48ff-a9dc-dc37ed9c9cb0)

After applying EXT PWR, it appears the AC and DC ESS BUS can't both be powered at the same time. They get stuck in an alternate state based on the true state of the ESS BAT from the previous step.

### Expected behavior

Electrical busses should be powered correctly based on the electrical bus logic. Aircraft should init with the electrical busses powered off when all ext power and battery are disconnected.

### Steps to reproduce

1. Load aircraft in 2024. Open the behaviour panel and inspect the electrical debug.
2. Observe the initial electrical state and observe how the state changes while connecting batteries and external power.

### Discord Username (optional)

Heclak

**Comment by MjrIncident** [comment]
Would this be the reason the radio does not auto tune like it does in 2020 - when using copilot auto atc?
Radio atc window says "no power...".

**Comment by heclak** [comment]
> Would this be the reason the radio does not auto tune like it does in 2020 - when using copilot auto atc?
> 
> Radio atc window says "no power...".

@MjrIncident Not sure about auto tuning but yes, comm 1 is on one of the affected busses thus it is unpowered in default ATC.

**Comment by heclak** [comment]
This was fixed with #9640.
